src/erlcloud_s3.erl

```diff
@@ -851,7 +863,7 @@ s3_request(Config, Method, Host, Path, Subresource, Params, POSTData, Headers) -
 s3_request(Config, Method, Host, Path, Subresources, Params, POSTData, Headers, GetOptions) ->
     {ContentMD5, ContentType, Body} =
         case POSTData of
```

Comment:
Ugh, I was wrong. I didn't notice there was another branch that match all in *the same line* .